scripts/performance/common.py

```diff
@@ -158,30 +158,39 @@ def push_dir(path: str = None) -> None:
     else:
         yield
 
-def retry_on_exception(function, retry_count=3, retry_delay=5, retry_delay_multiplier=1, retry_on_exception=Exception):
+def retry_on_exception(function, retry_count=3, retry_delay=5, retry_delay_multiplier=1, retry_exceptions=[Exception], raise_exceptions=[]):
     '''
     Retries the specified function if it throws an exception.
 
     :param function: The function to execute.
     :param retry_count: The number of times to retry the function.
     :param retry_delay: The delay between retries (seconds).
     :param retry_delay_multiplier: The multiplier to apply to the retry delay after failure.
-    :param retry_on_exception: The exception to retry on (Defaults to Exception).
+    :param retry_exceptions: The exception to retry on (Defaults to Exception). (Exclusive use from raise_exceptions)
```

Comment:
Can you update this to make it a bit more clear that you can't specify both `raise_exceptions` and `retry_exceptions` at the same time.

Reply:
Changed the verbiage: "Exclusive use from --_exceptions" -> "Cannot be used with --_exceptions"